tools/ColorTool/ColorTool/SchemeParsers/JsonParser.cs

```diff
@@ -1,18 +1,23 @@
-﻿using System;
+﻿//
+// Copyright (C) Microsoft.  All rights reserved.
+// Licensed under the terms described in the LICENSE file in the root of this project.
+//
+
+using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Runtime.Serialization.Json;
-using System.Text;
-using System.Threading.Tasks;
 using System.Xml;
 using static ColorTool.ConsoleAPI;
 
-namespace ColorTool
+namespace ColorTool.SchemeParsers
 {
     class JsonParser : ISchemeParser
     {
-        static string[] CONCFG_COLOR_NAMES = {
+        private const string FileExtension = ".json";
```

Comment:
>FileExtension [](start = 29, length = 13)

mind doing FILE_EXTENSION? I think this is about as const/static/readonly as the list below.